src/deployer.sol

```diff
@@ -30,7 +30,9 @@ contract LenderFabLike {
 }
 
 contract LenderLike {
+    function backer() public returns(address);
```

Comment:
I would split this into two interfaces because not all lenders implement backer()

Reply:
fixed